activesupport/lib/active_support/core_ext/date_and_time/calculations.rb

```diff
@@ -128,6 +128,17 @@ def beginning_of_year
     # The +given_day_in_next_week+ defaults to the beginning of the week
     # which is determined by +Date.beginning_of_week+ or +config.beginning_of_week+
     # when set. +DateTime+ objects have their time set to 0:00 unless +same_time+ is true.
+    #
+    # ==== Examples
+    #   today = Date.today
```

Comment:
we should be able to compress the example by putting the output on the same line:

```ruby
today = Date.today # => Thu, 07 May 2015
today.next_week    # => XYZ
```